hpgmgTest/Makefile

```diff
@@ -0,0 +1,14 @@
+HPGMG_ARCH = hpgmgTest
```

Comment:
These files shouldn't be added to the project -- they aren't relevant to other people.